houston/types.go

```diff
@@ -276,6 +279,7 @@ type AppConfig struct {
 	ConfigureDagDeployment bool   `json:"configureDagDeployment"`
 	NfsMountDagDeployment  bool   `json:"nfsMountDagDeployment"`
 	HardDeleteDeployment   bool   `json:"hardDeleteDeployment"`
+	PreCreatedNamespaces   bool   `json:"PreCreatedNamespaces"`
```

Comment:
the semantics of this var make me think its not a bool. I prefer name such as ShouldUsePreCreatedNamespaces for bools. Though it may be too annoying attack  this point to make that change (if it spans across repos)

Reply:
yea this flag is returned from houston